build.sbt

```diff
@@ -414,13 +414,13 @@ lazy val `api-tools` = (project in file("api-tools"))
   .enablePlugins(RuntimeLibPlugins)
   .settings(
     libraryDependencies ++= Seq(
-      "com.typesafe.play" %% "play-json" % PlayVersion,
+      "com.typesafe.play" %% "play" % PlayVersion,
       scalaTest % Test
```

Comment:
Yes, the api-tools uses some Play APIs which used to be brought in by the `server-javadsl` dependency, but since we removing that we had to explicitly depend on Play.